plugins/extractors/postgres/postgres.go

```diff
@@ -181,9 +178,14 @@ func (e *Extractor) getTableMetadata(db *sql.DB, dbName string, tableName string
 		return result, nil
 	}
 
+	host, err := splitURL(e.config)
```

Comment:
Parsing the URL connection could be done earlier, even it is possible to be parsed once the Extract function is called since this `host` value would be the same for all the metadata in this context. It is better to fail fast so user could catch and fix the error earlier.